lib/home/homepage-menus.tsx

```diff
@@ -8,6 +8,7 @@ import {
   OxygenIcon,
   DonationIcon,
 } from "~/components/ui/icons";
+import { ContributionIcon } from "~/components/ui/icons/contribution-icon";
```

Comment:
```suggestion
  DonationIcon,
  ContributionIcon,
} from "~/components/ui/icons";
```

Please include this icon inside the grouped exports on the `~/components/ui/icons/index.ts` file.